libethereum/ChainParams.cpp

```diff
@@ -116,8 +116,9 @@ ChainParams ChainParams::loadGenesis(string const& _json, h256 const& _stateRoot
 SealEngineFace* ChainParams::createSealEngine()
 {
 	SealEngineFace* ret = SealEngineRegistrar::create(sealEngineName);
-	assert(ret && "Seal engine not found");
-	ret->setChainParams(*this);
+	assert((ret != nullptr) && "Seal engine not found");
+	if (ret != nullptr)
```

Comment:
It is not explicit in the CodingStandards.txt, but it is used like that all over the codebase.
Pro: Much better readable, goes well with type changes
Con: Goes well with type changes